### DISCUSSION TITLE
최병호 블로그 1차 피드백 요청

### DISCUSSION THREAD

**Discussion by peppone-choi** [Orm-camp/blog-feedback#7] [동글동글 2기 피드백]
https://peppone-choi.github.io/posts/1%EC%B0%A8-%ED%94%84%EB%A1%9C%EC%A0%9D%ED%8A%B8-%ED%9A%8C%EA%B3%A0/

1차 프로젝트 회고 글을 썼습니다.

피드백 요청 드립니다 :)

**Comment by juuyeoon** [comment]
병호님, 임주연입니다! 프로젝트 회고글 작성하시느라 고생 많으셨어요 🙌🏻
회고를 하시며 내용을 기술해 주셨는데, 팀장 롤을 맡으시며 느끼거나 배우게 된 부분이 일반 팀원들에 비해 더 많으실 거라는 생각도 드네요!
그런 부분들을 천천히 글로 풀어주시면 어떨까 싶습니다.
그리고 Try로 다른 분들은 팀장을 맡으실 때 어떻게 하시나 체크해 보겠다고 하셨는데, 커뮤니티 등에서 소통하고 얻게 된 인사이트를 공유해 주시는 것도 정말 멋질 거 같네요 😊

**Reply by Yu-Kyeong-Seok** [reply]
글이 간결해서 보기 좋았습니다.
아쉬웠던거라면 좀 더 내용을 풀어서 친근감있게? 뭔가 보고서 같은 느낌이들어서 아쉬웠습니다!

**Comment by taeyoung0001** [comment]
카테고리를 클릭하면 그 해당 부분으로 이동하는 기능이 좋네요!

병호님은 팀장으로써 잘 주도하고 책임감있게 이끌어 나갔다고 생각합니다.

병호님만의 책임감을 절대 스스로 낮게 평가하지 않으셔도 될거같습니다 ㅎㅎ
늘 잘하고 계시잖아요!

**Comment by hyyun1113** [comment]
병호님! 블로그 글 잘 읽었습니다. 
전반적으로 간결한 문장으로 읽기 쉬웠으며, 팀장으로서 책임감을 느끼고 있는것이 잘 느껴졌습니다. 
다만, 구현기능은 조금 더 구체적으로 써도 좋지 않았을까 하는 생각이 듭니다. 간단한 기능들은 굳이 추가 하시지 않은걸로 생각되어지는데, 최종 결과물의 사진등 참고할 수 있는 자료도 없어서 글로라도 결과물의 형태를 상상할수있게끔 구체적으로 작성하면 더 좋지 않을까 하는 생각입니다. 
또한 후기에 팀 리더로써의 후기만을 작성해 주셨는데, 그 외에 '개발자'로서 이번 프로젝트를 하면서 느낀 후기도 추가한다면 어떨까요?

**Comment by dayoungh** [comment]
저도 주연님 피드백에 공감합니다. 팀장으로 어려웠던 부분이나 배우게 된 점, 팀원들을 서포트 해 준 부분들을 자세히 적어주신다면 병호님의 열정과 문제해결 능력을 글에서도  보여줄 수 있지 않을까요?😀 같은 조는 아니었지만 늘 솔선수범하시는 모습을 보면 프로젝트에서도 분명 그러셨을 거라고 생각합니다! 다음 글에서는 병호님의 능력을 마음껏 자랑해주시길ㅎㅎㅎㅎㅎ

**Comment by ggaong** [comment]
목차를 넣어 클릭 시 해당 부분으로 넘어가는 부분 너무 좋은 것 같아 저도 다음 글에는 참고해보고 싶네요!! 
다만 글을 읽다 보니 구현 부분에 대해 좀 더 자세히 작성해주면 어떨까 하는 생각이 들었습니다😄

글 작성하시느라 고생 많으셨습니다 🙌🙌

**Comment by euneunsol** [comment]
병호님! 회고글 잘 읽었습니다!
같은 팀원으로서 피드백을 드리자면..! 정말 많이 고생하셨고 정말 많이 도와주셨는데
그런 부분이 빠진것 같아 아쉬워요! 
구현 기능 부분을 조금 더 자세하게 작성하셔서 이거 다 제가 했어요! 라고 자랑한번 해주세요~! :)
